sites/feedback/index.json-ld

```diff
@@ -969,10 +973,15 @@
 	],
 	"iteration": [
 		{
-			"@id": "_:iteration_pft_1",
-			"name": "Page feedback tool - Iteration 1",
-			"date": "2023-08",
-			"detectableBy": ".gc-pft",
+			"@id": "_:iteration_pft_2",
+			"name": "Page feedback tool - Iteration 1.1",
+			"date": "2024-07",
+			"detectableBy": "#gc-pft:has(.gc-pft-no p[role=\"status\"]:nth-of-type(2))",
+			"fixes": [
+				"AJAX fragment: Added <code>pageData</code> to the French variant",
+				"AJAX fragment: Changed <code>aria-live=\"polite\"</code> to <code>role=\"status\"</code> in \"Tell us why below:\"."
+			],
```

Comment:
Are the "AJAX fragment" prefixes appropriate?

Are there any standardized prefixes these kinds of notes are supposed to be following?

Here's what I've spotted:
* Some pre-existing ``fixes`` arrays contain no prefixes at all, whereas others use "Demoted/Style:".
* The similar-looking ``breaking``/``additions`` arrays are usually prefixed with "Layout/Semantic/Behaviour:"... but sometimes lack prefixes.